README.md

```diff
@@ -98,7 +102,7 @@ before flagging as acceptable for merging into the gem5 website repository.
 polite manner. Rude and/or dismissive remarks will not be tolerated**.
 
 Once your change has been accepted by reviewers you will be able to click
-`Submit` within your changes Gerrit page. This focally merges the change
+`Submit` at the bottom of your pull request. This focally merges the change
```

Comment:
Just an FYI: The purpose of this pull-request is just to get Melissa's changes from Gerrit into the Repo. I intend on doing a more thorough go through of the README's, etc. in another pull request. I wanted to get this in first to avoid any conflicting messes.